README.md

```diff
@@ -36,10 +36,13 @@ The last one can be run by:
   - users in the `quizzo-manage-secrets` group
   - programs on ec2 instances which have been started with the `quizzo-secrets` IAM profile
 
-To start an instance with the quizzo-secrets IAM profile from the CLI:
+To start an EC2 instance with the quizzo-secrets IAM profile from the CLI:
 
   `aws ec2 run-instances ...--iam-instance-profile Name=quizzo-secrets`
 
+To start an ECS cluster with the quizzo IAM profile, select quizzo-secrets-instances from the
+Container Instance IAM Role selection on the Create Cluster screen.
```

Comment:
I tried the following steps:
```
aws ecs create-cluster --cluster-name quizzo
instance=...# get the public ip
ssh $instance curl http://169.254.169.254/latest/dynamic/instance-identity/document/ > identity-doc.json
ssh $instance curl http://169.254.169.254/latest/dynamic/instance-identity/signature/ > identity-sig.txt
aws ecs register-container-instance --cluster quizzo  \
       --instance-identity-document file://identity-doc.json \
       --instance-identity-document-signature file://identity-sig.txt
```
Which seems to be the documented way to start an instance on a cluster for a container.  However, I received this message:
```
An error occurred (ServerException) when calling the RegisterContainerInstance operation: Service Unavailable. Please try again later.
```
As far as I can see, this is the documented way of doing it.  Whether or not the aws api works is another question.